## Appendix

### Bioinformatics

#### Databases

Bioinformatics deals with biological data, their collection, curation, distribution, and analysis. The ‘unit’ of distribution of a collection of some type of biological information is a database.

A database includes (1) an archive of information, (2) a logical organization or ‘structure’ of that information, called a schema, and (3) tools to gain access to it.

Types of databases:

- Bibliographic databases
- Databases of websites
- Archival databases of biological information
- Derived Databases

#### Neuroscience databases

For an extensive list of Neuroscience databases check the [Wiki](https://en.wikipedia.org/wiki/List_of_neuroscience_databases).

**PsyArXiv**

**PsyArXiv** is a preprint repository for the psychological sciences.

**Neuroimaging and Neurophysiology Database:**

[https://openneuro.org/](https://openneuro.org/)

**Neuron Morphology**

[https://neuromorpho.org/](https://neuromorpho.org/)

**Machine Learning**

Neural Network Dataset:

MNIST Dataset:

[http://yann.lecun.com/exdb/mnist/](http://yann.lecun.com/exdb/mnist/)

The same dataset but in CSV ([[#^footnote-47|Comment Unknown Author 47]]) format:

[http://pjreddie.com/projects/mnist-in-csv/](http://pjreddie.com/projects/mnist-in-csv/)

#### Biology Databases

[**NCBI**](https://www.ncbi.nlm.nih.gov/)

**Pubmed**

[https://pubmed.ncbi.nlm.nih.gov/](https://pubmed.ncbi.nlm.nih.gov/)

PMID – PubMed reference number

**GenBank**

NIH genetic sequence database, an annotated collection of all publicly available DNA sequences.

**The Human Gene Mutation Database (HGMD®)** - Constitutes a comprehensive collection of published germline mutations in nuclear genes that underlie or are closely associated with human inherited disease. [http://www.hgmd.org/](http://www.hgmd.org/) (Out of date)

Utilizes Entrez retrieval system.

[**OMIM**](https://www.omim.org/)

An Online Catalog of Human Genes and Genetic Disorders.

**Encode**

[https://www.encodeproject.org/](https://www.encodeproject.org/)

The goal of the ENCODE project (derived from Encylopaedia of DNA Elements) is to understand the function of the entire human genome.

**Expasy**

[https://www.expasy.org/](https://www.expasy.org/)

It contains (search bar):

**Translate** - Translation of a nucleotide (DNA/RNA) sequence to a protein sequence.

**UniProtKB** - The UniProt Knowledgebase is a central hub for the collection of functional information on proteins with accurate, consistent, and rich annotation.

It contains:

BLAST and alignment tools.

**GENE ONTOLOGY**

[http://geneontology.org/](http://geneontology.org/)

The Gene Ontology (GO) knowledgebase is the world’s largest source of information on the functions of genes.

**Gene Cards**

[https://www.genecards.org/](https://www.genecards.org/)

Gene Cards is a searchable, integrative database that provides comprehensive, user-friendly information on all annotated and predicted human genes. The knowledgebase automatically integrates gene-centric data from ~150 web sources, including genomic, transcriptomic, proteomic, genetic, clinical, and functional information.

**Pfam**

[http://pfam.xfam.org/](http://pfam.xfam.org/)

The Pfam database is a large collection of protein families, each represented by multiple sequence alignments and hidden Markov models (HMMs). Needs ID from UniProt.

**Others**

**MEROPS** – Protease information

**Protein Kinase Resource (KinaseNET)** – Kinase information

**InterPro** - Integrates the contents, features, and annotation of several individual databases of protein families, domains, and functional sites, and contains links to others, including the Gene Ontology Consortium functional classification.

**Molecular interaction databases:**

**IntAct**

[https://www.ebi.ac.uk/intact/home](https://www.ebi.ac.uk/intact/home)

IntAct provides a free, open-source database system and analysis tools for molecular interaction data.

**Full-genome sequencing projects databases:**

**Ensembl** - Homo Sapiens

[https://www.ensembl.org/index.html](https://www.ensembl.org/index.html)

**FlyBase** – Other organisms

[https://flybase.org/](https://flybase.org/)

**GISAID** – SARS-CoV-2

[https://www.gisaid.org/](https://www.gisaid.org/)

**Nucleotide sequence databases:**

**European Nucleotide Archive (ENA)**

[https://www.ebi.ac.uk/ena/browser/home](https://www.ebi.ac.uk/ena/browser/home)

**Micro-RNA databases:**

**miRBase**

[http://www.mirbase.org/](http://www.mirbase.org/)

**Others**

[http://www.cuilab.cn/hmdd](http://www.cuilab.cn/hmdd)

(Ligação de microRNAs a doenças)

[https://mircarta.cs.uni-saarland.de/](https://mircarta.cs.uni-saarland.de/)

(Informação geral sobre microRNAs)

[https://mirna.wustl.edu/](https://mirna.wustl.edu/)

(Dados de expressão de microRNAs no CNS)

[https://ccb-web.cs.uni-saarland.de/tissueatlas/](https://ccb-web.cs.uni-saarland.de/tissueatlas/)

(Dados de expressão de microRNAs)

[http://mips.helmholtz-muenchen.de/mitalos/#/search](http://mips.helmholtz-muenchen.de/mitalos/#/search)

(Alvos de microRNAs e vias metabólicas/sinalização)

**DDBJ Center**

[https://www.ddbj.nig.ac.jp/index-e.html](https://www.ddbj.nig.ac.jp/index-e.html)

**Tools for selection and retrieval of sequences:**

**EMBL-EBI**

[https://www.ebi.ac.uk/](https://www.ebi.ac.uk/)

It contains a BLAST tool.

[https://www.ebi.ac.uk/Tools/sss/ncbiblast/nucleotide.html](https://www.ebi.ac.uk/Tools/sss/ncbiblast/nucleotide.html)

ENTREZ (NCBI)

**Rare disease databases:**

**Orphanet** - Orphanet is a unique resource, gathering and improving knowledge on rare diseases so as to improve the diagnosis, care, and treatment of patients with rare diseases.

[https://www.orpha.net/consor/cgi-bin/index.php?lng=EN](https://www.orpha.net/consor/cgi-bin/index.php?lng=EN)

**Bibliographical:**

**Google Scholar**

[https://scholar.google.com/](https://scholar.google.com/)

**Pirate bibliographic database:**

Sci-hub

[https://sci-hub.hkvisa.net/](https://sci-hub.hkvisa.net/)

**Base de dados portuguesa:**

**MSGP**

[https://msgp.pt/](https://msgp.pt/)

The main goal of this catalogue is to curate all the proteins described as being part of stress granules in mammalian cells. Moreover, it also identifies different features of stress granules components, and importantly their expression profile in the context of neurodegenerative diseases such as Alzheimer’s, Huntington’s, and Parkinson’s diseases.

**Biblioteca do conhecimento online**

[https://www.b-on.pt/](https://www.b-on.pt/)

**Sapientia (Ualg)** – Dissertações e monografias.

[https://sapientia.ualg.pt/](https://sapientia.ualg.pt/)

Fármacos – INFOMED

[https://extranet.infarmed.pt/INFOMED-fo/](https://extranet.infarmed.pt/INFOMED-fo/)